linters/.eslintrc

```diff
@@ -152,7 +152,7 @@
     "no-new-object": 2,              // http://eslint.org/docs/rules/no-new-object
     "no-spaced-func": 2,             // http://eslint.org/docs/rules/no-spaced-func
     "no-trailing-spaces": 2,         // http://eslint.org/docs/rules/no-trailing-spaces
-    "no-wrap-func": 2,               // http://eslint.org/docs/rules/no-wrap-func
+    "no-extra-parens": 2,            // http://eslint.org/docs/rules/no-extra-parens
```

Comment:
I strongly prefer using "extra" parens with boolean logic - ie, `a && (b || c)`, solely for clarity. This rule, without configuration, warns on this.